memcachepp/memcache/detail/handle_access.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright 2015 (c) Bill <billtian945@gmail.com>
+// Distributed under the Boost Software License, Version 1.0.
+// (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef __MEMCACHE_DETAIL_HANDLE_ACCESS_HPP__
```

Comment:
Please do not use "__MEMCACHE_..." as the standard states that identifiers or tokens starting with double underscore are reserved. Use "MEMCACHE_..." instead.